metrics/engine/ingester.go

```diff
@@ -4,10 +4,14 @@ import (
 	"time"
 
 	"github.com/sirupsen/logrus"
+	"go.k6.io/k6/metrics"
 	"go.k6.io/k6/output"
 )
 
-const collectRate = 50 * time.Millisecond
+const (
+	collectRate          = 50 * time.Millisecond
```

Comment:
This seems to not be used as well

Reply:
https://github.com/grafana/k6/blob/d2db8522f0b300e720215002bd31221799448119/metrics/engine/ingester.go#L38

Reply:
uff, sorry was looking it is new and then searched it in the PR code :facepalm: ... but it is not new :facepalm: